josuke.go

```diff
@@ -40,10 +41,9 @@ func parseLogLevel(value string) (LogLevel, bool) {
 // Josuke is the main object, that contains the HTTP server configuration
 // and the hook definitions.
 type Josuke struct {
-	LogLevelName string   `json:"logLevel"`
+	LogLevelName string `json:"logLevel"`

```

Comment:
Format nitpicking, not aligned with the below options.

Reply:
This is how `gofmt` works, dunno if this is intended tho. The line without annotation below creates a separation in the indenting mechanism of the whole structure.
Will move the unannotated line for Adrian Monk' sake